application/forms/AddNodeForm.php

```diff
@@ -464,8 +539,18 @@ public function onSuccess()
     {
         $changes = ProcessChanges::construct($this->bp, $this->session);
         switch ($this->getValue('node_type')) {
-            case 'host':
             case 'service':
+                if ($this->statesToString() != null) {
```

Comment:
`!==` :wink:

Reply:
I Raplaced it with:

```suggestion
                if ($this->getValue($stateName) !== null && $this->getValue($stateName) !== '') {
```
beacase, if not set it returns an emty string, but for the OK => 0 state i dont creata an "Override State" Dropdown menu. So that returns `null`